frontend/src/components/pages/CampCreation/RegistrationForm/index.tsx

```diff
@@ -62,6 +65,12 @@ const RegistrationFormPage = ({
     onClose: onAddQuestionClose,
   } = useDisclosure();
 
+  const filteredFixedCamperInfoQuestions = campOffersEDLP
+    ? fixedCamperInfoQuestions
+    : fixedCamperInfoQuestions.filter(
+        (question) => question.question !== EDLP_CAMPER_INFO_QUESTION,
+      );
```

Comment:
nit: Rather than removing it every time, and having to filter through an array, can we just reverse the logic and add the question if relevant?

```ts
  const filteredFixedCamperInfoQuestions = campOffersEDLP
    ? fixedCamperInfoQuestions.push(EDLP_CAMPER_INFO_QUESTION);
    : fixedCamperInfoQuestions;
```